src/main/java/core/basesyntax/serviceimpl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.storage.Storage;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
There is an unnecessary empty line at the beginning of the class definition. According to the checklist, you should remove it.